lib.php

```diff
@@ -998,22 +964,23 @@ function format($string)
  */
 function block_coursefeedback_adminurl($mode, $action, $fid = null, array $other = array())
 {
-	$url = new moodle_url("/blocks/coursefeedback/admin.php");
-	$params = array_merge($other, array("mode" => $mode, "action" => $action));
-	if (is_number($fid))
-		$params["fid"] = $fid;
-	$url->params($params);
+    $url = new moodle_url("/blocks/coursefeedback/admin.php");
+    $params = array_merge($other, array("mode" => $mode, "action" => $action));
+    if (is_number($fid))
+        $params["fid"] = $fid;
+    $url->params($params);
 
-	return $url;
+    return $url;
 }
 
 /**
  * Checks if a Feedbackperiod is active.
  *
  * @return array|bool $period if active, false if not, true if no periods given
  */
-function block_coursefeedback_period_is_active() {
-    $config     = get_config("block_coursefeedback");
+function block_coursefeedback_period_is_active()
+{
+    $config = get_config("block_coursefeedback");
```

Comment:
Wieso machst du unterschiedliche Coding styles für IF und FUNCTION? Zuerst dachte ich die Zeilenumbrüche bei FUNCTION hast du übersehen, aber offensichtlich ist das eine gewollte Entscheidung, die für mich nicht nachvollziehbar ist. Daher: bitte einheitlich. Entweder machst du vor den geschweiften Klammer immer einen Zeilenunbruch, oder nirgends.

Reply:
Grad nochmal nachgelesen: Moodle Coding Style ist geschweifte Klammer in der gleichen Zeile wie das FUNCTION Schlüsselwort. Also machen wir das so.

Reply:
```suggestion
function block_coursefeedback_period_is_active() {
    $config = get_config("block_coursefeedback");
```

Reply:
Ja hab das auf dem Schirm (Martin hatte das auch schon angemerkt), diese Coding style Änderungen würde ich dann alle in einen Commit und PR packen ;)

Reply:
Ich hatte mich mit den geschweiften Klammern teilweise an dem alten Code orientiert (dort gibt es teilweise die neue Zeile für die geschweiften Klammern) und teilweise hab ich es so gemacht wie gewohnt. 
Ich werde das in einem Commit ändern.

Reply:
Dann kann ich diesen PR ja schließen... nicht wahr?